test/approvals/_dcuc_smbios_data_work_novacustom_nv4x_adl_v1_6_0_rom_s_D07229051

```diff
@@ -0,0 +1,3 @@
+Will modify /home/macpijan/projects/github/dasharo/dcu/test/data/work/novacustom_nv4x_adl_v1.6.0.rom
+
+Configurable Serial Number not supported by the coreboot image
```

Comment:
Can't reproduce that. I get 
`Failed to extract coreboot configuration from the image`

Reply:
Please add `set -x` at the top of `dcu` file and try to debug this more